ncm-network/src/main/perl/network.pm

```diff
@@ -279,7 +279,7 @@ sub Configure
                     $text .= "OVS_OPTIONS='$net{$iface}{ovs_opts}'\n";
```

Comment:
also here? if we would do this for the whole component, we better have a TT file create this. but this will trigger a mass network restart